src/test/java/seedu/anilist/model/anime/StatusEqualsPredicateTest.java

```diff
@@ -13,31 +16,31 @@
     @Test
     public void equals() {
         Status firstStatus = new Status(VALID_STATUS_TOWATCH);
-        Status secondStatus = new Status(VALID_STATUS_WATCHING);
+        Status secondStatus = new Status(VALID_STATUS_FINISHED);
```

Comment:
why this change lol?

Reply:
um 'watching' is the default status, so this change is to check if changing this to a non default status would affect anything.